mev-build-rs/src/payload/builder.rs

```diff
@@ -95,10 +96,26 @@ fn append_payment<Client: StateProviderFactory>(
 
     let signer_account = db.load_cache_account(signer.address())?;
     let nonce = signer_account.account_info().map(|info| info.nonce).unwrap_or_default();
+
+    let proposer_fee_recipient_account = db.load_cache_account(config.proposer_fee_recipient)?;
+    let is_empty_code_hash =
+        proposer_fee_recipient_account.account_info().unwrap_or_default().is_empty_code_hash();
```

Comment:
this is very minor, but I would guess it is cheaper to unwrap at the end

as written, it would construct a full `AccountInfo::default()` instance, and then call `is_empty_code_hash`

if we do this, it would skip the instantiation if the account is missing

```suggestion
        proposer_fee_recipient_account.account_info().map(|account| account.is_empty_code_hash()).unwrap_or_default();
```